pkg/api/controller.go

```diff
@@ -49,6 +50,18 @@ func (c *Controller) Run() error {
 		os.Exit(1)
 	}
 
+	// Updating the CVE Database
+	if c.Config != nil && c.Config.Extensions != nil && c.Config.Extensions.Search != nil &&
+		c.Config.Extensions.Search.CVE.UpdateInterval > 1 {
+		go func() {
+			err := cveinfo.UpdateCVEDb(c.Config.Storage.RootDirectory, c.Log,
+				c.Config.Extensions.Search.CVE.UpdateInterval, false)
+			if err != nil {
```

Comment:
Add a log to indicate that an update has started.
Add a log in this path for errors.
Also add a log that update was successful etc, so we know what is going on.

Reply:
There is a log in UpdateCVEDb method, I will add more for more clarifications.

Reply:
But we want to track when this goroutine got invoked and is done.

Reply:
Yes, i will add the log for that